main/input_default.cpp

```diff
@@ -111,6 +111,12 @@ bool InputDefault::is_action_pressed(const StringName &p_action, bool p_exact) c
 }
 
 bool InputDefault::is_action_just_pressed(const StringName &p_action, bool p_exact) const {
+#ifdef TOOLS_ENABLED
+	if (_currently_parsing_input) {
+		WARN_PRINT_ONCE("Prefer InputEvent.is_action_pressed() within _input() to prevent detecting duplicates.");
```

Comment:
```suggestion
		WARN_PRINT_ONCE("Prefer InputEvent.is_action_pressed() within input event callbacks to prevent detecting duplicates.");
```

Maybe something like this as `_gui_input()` and `_unhandled_input()` may have this problem too.

Reply:
Done. :+1: 
The wording here I found quite tricky as it has to be concise but convey some sense of the problem and solution.